### PR TITLE
feat(app): run newly created spec

### PR DESCRIPTION
**Pull request by lmiller1990**
https://cypress-io.atlassian.net/browse/UNIFY-142

### CODE REVIEW

File: packages/data-context/src/actions/ProjectActions.ts [hunk 1]
```diff
@@ -435,13 +438,18 @@ export class ProjectActions {
     const [newSpec] = codeGenResults.files
 
     const spec = this.ctx.file.normalizeFileToSpec({
-      absolute: newSpec.file,
+      absolute: path.join(project.projectRoot, newSpec.file),
```
Comment (ZachJW34) [inline packages/data-context/src/actions/ProjectActions.ts:441]: The `newSpec.file` should be an absolute path. Looks like the `packages/app/src/specs/generators/component/ComponentGeneratorStepOne.vue` component is passing in a relative path to the mutation when it should be an absolute, which is why you had to add the line above.

**Comment by ZachJW34** [comment]
Changes look good. I ran into the same issue with e2e tests not re-querying data when it works when doing normal dev.

File: packages/data-context/src/actions/ProjectActions.ts (hunk 1 above)
Comment (lmiller1990) [reply packages/data-context/src/actions/ProjectActions.ts]: Yep, you are right. Fixed.

**Comment by lmiller1990** [comment]
This is basically done, but we need the new `specPattern` stuff done to fully complete this feature - getting the correct testFiles pattern is really difficult with how we have things set up. I'll complete that before merging this up.

**Pull request closed without merging**